modules/issue_tracker/README.md

```diff
@@ -1,10 +1,10 @@
-## Issue Tracker Readme
+# Issue Tracker
 
-#### Overview
+## Purpose
 The Issues Module allows users to track issues they have with data, or with their LORIS instance itself. A form with pre-defined fields is provided for users to submit issues, and a filter-form gives a sortable and filterable table view of issues viewable by the user.
 
-#### Permissions
-Permissions in this module are complex and are not yet in their final iteration. Two new permissions were created, issue_tracker_reporter and issue_tracker_developer. The view_all_sites permission was also integrated to prevent users from viewing issues relevant to data they do not have permission to see. A reporter can add an issue, edit their own issue, and comment on all issues. A developer can do the same, plus they can close an issue and edit all fields of a submitted issue. Most of the permissions are controlled in IssueForm.js, dependant on values returned in editIssue.php.
+## Permissions
+Permissions in this module are complex and are not yet in their final iteration. Two new permissions were created, `issue_tracker_reporter` and `issue_tracker_developer`. The `view_all_sites` permission was also integrated to prevent users from viewing issues relevant to data they do not have permission to see. A reporter can add an issue, edit their own issue, and comment on all issues. A developer can do the same, plus they can close an issue and edit all fields of a submitted issue. Most of the permissions are controlled in `IssueForm.js`, dependent on values returned in `editIssue.php`.
```

Comment:
Not sure it is a good idea to say "... are complex and not yet in their final iteration". => Better not use it for now or don't open this can of worms might come to user mind.

So, the "view_all_sites" allow you to view what your are not supposed to? Maybe say that this permission is for users who need to see all issues even those related to data they can't see. But wait, if the text of the issue have include some of the data, that would equate to data leak?

Are there user who are developper? (I mean in a production environnement.)

Line about the code (js and PHP files) is irrelevant to end users

Reply:
```suggestion
Two new permissions were created, `issue_tracker_reporter` and `issue_tracker_developer`. The `view_all_sites` permission was also integrated allow *** WHAT TYPE OF USER EXACTLY FOR HERE?*** viewing issues relevant to data from others sites. A reporter can add an issue, edit their own issue, and comment on all issues. A developer can do the same, plus they can close an issue and edit all fields of a submitted issue. Most of the permissions are controlled in `IssueForm.js`, dependent on values returned in `editIssue.php`.
```